### DISCUSSION TITLE
<title>Using git bash diff color display problem

### DISCUSSION THREAD

**Discussion by digger-yu** [community/community#53589] [Other Feature Feedback, Questions, & Ideas] [answered]
when i use git bash "git diff --cached"
the result like this 
![image](https://user-images.githubusercontent.com/55081697/233931955-377c3556-8677-42d0-960b-caa7b4ad7ffc.png)

How to set to show only the word color of the diff, not the line color

**Comment by airtower-luna** [comment, marked as answer]
Maybe you mean the [`--color-words` option](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---color-wordsltregexgt)?

**Reply by digger-yu** [reply]
This is it, thank you very much!
